### DISCUSSION TITLE
How to use PrimaryScrollController in HookWidget

### DISCUSSION THREAD

**Discussion by dodatw** [rrousselGit/flutter_hooks#339] [Q&A] [answered]
I am using scrollControll by
```
    final scrollController = useScrollController()
```

But I have a case that I need use PrimaryScrollController. (tap status bar to trigger scroll-to-top)
```
    final scrollController = PrimaryScrollController.of(context);
```

Can it be used in HookWidget? Is there any best practice for this case ?

**Comment by rrousselGit** [comment, marked as answer]
There shouldn't be an issue with using `PrimaryScrollController.of` inside HookWidgets.
